helm-charts/splunk-otel-collector/templates/config/_otel-k8s-cluster-receiver-config.tpl

```diff
@@ -35,6 +35,11 @@ receivers:
     {{- if eq (include "splunk-otel-collector.distribution" .) "openshift" }}
     distribution: openshift
     {{- end }}
+  {{- if and $clusterReceiver.objectsEnabled "true" (eq (include "splunk-otel-collector.logsEnabled" .) "true") }}
+  k8sobjects:
+    auth_type: serviceAccount
+    objects: {{ .Values.clusterReceiver.k8sObjects | toJson }}
```

Comment:
Should this use `toYaml`? It would probably be worth adding a new manifest run w/ values that exercise this to help vet the output.

Reply:
I'm not sure so would defer to whatever makes sense to your use case. thanks for adding the rendered manifests.